bin/release

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+source ./venv/bin/activate
+
+set -ux
+rm -r ./dist/
+git fetch yelp
+git checkout yelp/stable -B stable
+git tag -f v"$(python venv_update.py --version)"
+git push yelp --tags
+python setup.py sdist bdist_wheel
+python setup.py register
```

Comment:
It's idempotent and quick.
It covers the "if you need to set up pypi auth" of the old `.md`.